src/components/DatasetEditor/DatasetEditor.test.tsx

```diff
@@ -159,6 +159,30 @@ describe('Dataset Editor', () => {
     expect(selectors.item(true, 'A:Value')).not.toBeInTheDocument();
   });
 
+  it('Should not allow selecting already selected fields', () => {
+    const { value, onChange } = createOnChangeHandler([{ name: 'Time', source: 'A' }]);
+
+    /**
+     * Render
+     */
+    render(
+      getComponent({
+        value,
+        onChange,
+      })
+    );
+
+    const newItemNameInput = selectors.newItemName();
```

Comment:
variable is not needed here because selector is used once

Reply:
fixed